src/main/java/org/embulk/input/FtpFileInputPlugin.java

```diff
@@ -66,7 +66,7 @@
         public String getHost();
 
         @Config("port")
-        @ConfigDefault("null")
+        @ConfigDefault("21")
```

Comment:
I moved into [newFTPClient()](https://github.com/embulk/embulk-input-ftp/pull/5/commits/14b686b5eb03d79fa8088c8e838a25f69a7c4522#diff-6c1d7324b8325e41503f63c5192e8ee4R182) and changed default value as follows.
* FTP 21
* FTPES 21
* FTPS 990